test/extended/testdata/bindata.go

```diff
@@ -34135,6 +34137,42 @@ func testExtendedTestdataCmdTestCmdTestdataApplicationTemplateStibuildJson() (*a
 	return a, nil
 }
 
+var _testExtendedTestdataCmdTestCmdTestdataExternalServiceIpv6Yaml = []byte(`apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    external: service
+  name: external
+  resourceVersion: "1"
+  uid: 19cff995-5546-11e5-9f57-080027c5bfa9
+spec:
+  clusterIP: fd02::1
```

Comment:
fd02::1 is already taken, I'd pick something else a bit higher.

Reply:
fd02::8 and fd02::9 should work.